src/app/static/highlighted-response/highlighted-response.component.html

```diff
@@ -2,9 +2,9 @@
   <div>
     HIGHLIGHTED RESPONSE
   </div>
-  <mat-card-header>
+  <mat-card-header> 
     <mat-card-subtitle
-      >{{ responderName }},
+      >{{ this.highlight.responderId }},
```

Comment:
Why change this to id? It looks like you could grab the name the same way as the prior version, and the name is better for users

Reply:
This was a mistake, I just wanted to remove extra variables.